src/components/SQForm/SQFormResetInitialValuesButton.tsx

```diff
@@ -13,14 +32,15 @@ function SQFormResetInitialValuesButton({
   tooltip = 'Reset Button',
   initialValuesObject = {},
   ...props
-}) {
+}: Props): JSX.Element {
   const {isDialogOpen, openDialog, closeDialog} = useDialog();
-  const {values, resetForm} = useFormButton(isDisabled);
+  const {values, resetForm} = useFormButton(isDisabled, false);
 
-  const handlePrimaryButtonClick = () => {
-    resetForm({
+  const handlePrimaryButtonClick = (): void => {
+    //TODO: Remove ResetForm and FormikValues once useFormButton is TSified
```

Comment:
`useFormButton` has already been converted to TS, but looks like its types can be improved

Reply:
#382

Reply:
Merged #382. Pull that change in and see if the types are working

Reply:
Sure thanks